File: Easy/3658. GCD of Odd and Even Sums/solution.cs
public class Solution {
    public int GcdOfOddEvenSums(int n) {
        int sumOdd = n * n;
        int sumEven = n * (n + 1);

        return GCD(sumOdd, sumEven);
    }

    public int GCD(int a, int b){
        while(b > 0){
            int tmp = b;
            b = a % b;
            a = tmp;
        }
        return a;
    }
}

File: Easy/3658. GCD of Odd and Even Sums/solution.py
class Solution:
    def gcdOfOddEvenSums(self, n: int) -> int:
        sumOdd = n * n
        sumEven = n * (n + 1)

        def gcd(a, b):
            while b > 0:
                tmp = b
                b = a % b
                a = tmp
            return a
        return gcd(sumOdd, sumEven)
